test/Microsoft.SqlTools.ServiceLayer.IntegrationTests/DacFx/DacFxServiceTests.cs

```diff
@@ -859,15 +856,7 @@ private bool ValidateOptions(DeploymentOptions expected, DeploymentOptions actua
                 else
                 {
                     //Verifying expected and actual deployment options properties are equal
-                    Assert.True((defaultP == null && actualP == null) 
```

Comment:
Removing defaultP/actualP and only checking defaultPValue/actualPValue was speculation/a question on my part; can you confirm that defaultP/actualP are no longer important to check?

Reply:
We do not require now, as the defaultPValue get the value or null if defaulp (line 848), checking defaulPvalue is same here, and actuyalPValue is same it will be either null or "" depending on actualP which we do not needed in assertion. Thanks.